### DISCUSSION TITLE
Error exporting metrics

### DISCUSSION THREAD

**Discussion by ImranDJCS** [DaxStudio/DaxStudio#617] [Q&A]
Hi there,
In DAX Studio, I'm connected with my Power BI file and can see model in the left pane. When I click on Advanced > View Metrics it gives error in Output window with a column name which I can't find in my model (in file or in Dax Studio). Could anyone advise how to locate that particular column?

![Export Metrics - error](https://user-images.githubusercontent.com/83199486/117751066-d0358780-b257-11eb-8d97-e9b89ebfdede.PNG)

**Comment by dgosbell** [comment]
At a guess it's most likely a reference in one of your measures or something like that. Can you try running something like 

```
SELECT [Name],[Expression] 
FROM $SYSTEM.TMSCHEMA_MEASURES
```

then export the results to a csv file and then search it using a text editor. Or copy/paste them into Excel and search there. Unfortunately the DMV syntax does not support wildcard search queries.

**Reply by ImranDJCS** [reply]
I find out it was not a referenced column in DAX or something, but a new column added in DB for an entity in Power BI Model.
Its fixed now. :)
Thank you
